source/com.google.crypto.tink/tink-android/Transforms/Metadata.xml

```diff
@@ -819,4 +819,102 @@
     </add-node>
 
 
+
+    <add-node
+        path="/api/package[@name='com.google.crypto.tink.signature']/class[@name='RsaSsaPssPrivateKey']"
+        >
+        <method 
+            visibility="public" static="false" abstract="false" return="com.google.crypto.tink.Key" name="getPublicKey"
+            deprecated="not deprecated" final="false" bridge="false" native="false" 
+            synchronized="false" synthetic="false" 
+            />
+    </add-node>
+    <attr
+        path="/api/package[@name='com.google.crypto.tink.signature']/class[@name='RsaSsaPssPrivateKey']"
```

Comment:
I don't think `propertyName` is valid for a `class`.

Reply:
True. Fixed. Thanks